packages/node_modules/pouchdb-security/lib/index.js

```diff
@@ -185,7 +185,7 @@ var requiresMemberWrapper = securityWrapper.bind(null, function (userCtx, securi
 
 [].concat(
   "get allDocs getAttachment info revsDiff getSecurity list".split(" "),
-  "show update rewriteResultRequestObject".split(" ")
+  "show update rewriteResultRequestObject bulkGet".split(" ")
```

Comment:
Out of curiosity, why the `[].concat('string string'.split(' '))` instead of just `['string', 'string']`?

Reply:
Originally? Laziness. Now? Consistency. We could change it, but that should be a separate PR I think.